classes/PodsInit.php

```diff
@@ -822,6 +824,28 @@ public function register_assets() {
 
 		wp_set_script_translations( 'pods-dfv', 'pods' );
 
+		$config = [
```

Comment:
:no_entry_sign: Short array syntax is not allowed

<sub>hash: 844dac133e3e00fd53ce4c808bc34737</sub>